ext/standard/var_unserializer.re

```diff
@@ -31,6 +31,8 @@
 #define VAR_WAKEUP_FLAG 1
 #define VAR_UNSERIALIZE_FLAG 2
 
+#define DEFAULT_MAX_DEPTH 4096
```

Comment:
Making this configurable `-d somedirective.unserialize_default_max_depth` might be convenient for non-technical users that see this error don't want to patch/fork old third-party libraries that deal with heavily nested data. (e.g. it can be overridden, but only to increase it to >= 4096 or to infinity(0))

(similar to what xdebug.max_nesting_level does for method recursion)

- This might just be hypothetical, I haven't seen any applications like that, although things such as AST/CFG parsers (any language) with `1 + 1 + 1 + 1 + 1` could create trees of binary operations with thousands of nodes